.github/workflows/deploy.yml

```diff
@@ -7,8 +7,9 @@ on:
       - '**/README.md'
 
 jobs:
+
   test_and_deploy:
-    uses: communitiesuk/funding-design-service-workflows/.github/workflows/deploy.yml@main
+    uses: communitiesuk/funding-design-service-workflows/.github/workflows/deploy.yml@postgres
```

Comment:
I'd rather we didn't depend on a branch of the shared workflow. Can we either:
* Add a flag to the main workflow to support whatever this is doing 
or
* fork and not use the shared workflow?

Reply:
I would prefer if this was pointing to main, or forks the workflow before this is merged.

Reply:
@gio-karunakaran @SentientHousePlant this is still not resolved as we are still pointing to a branch of the shared workflow.